test/unit/puppetlabs/services/jruby/jruby_service_test.clj

```diff
@@ -31,7 +62,7 @@
        (try
          (bootstrap/with-app-with-config
           app
-          jruby-testutils/default-services
+          (conj jruby-testutils/default-services jruby-pooled-test-service)
```

Comment:
Oh, I get it now.  You needed this because previously the default service list had a service like this in it, and it doesn't anymore.  This seems fine for now.